index.js

```diff
@@ -61,7 +62,7 @@ module.exports = {
           : [],
         APPLITOOLS_CONCURRENCY: inputs.concurrency,
         PAGES_TO_CHECK: builtPages,
-        CYPRESS_CACHE_FOLDER: path.resolve(PUBLISH_DIR, '..', 'node_modules'),
+        CYPRESS_CACHE_FOLDER: './node_modules/CypressBinary',
```

Comment:
Same reason as before to use `node_modules`.
See also https://github.com/cypress-io/netlify-plugin-cypress#recommended